odo/backends/tests/test_s3.py

```diff
@@ -93,6 +93,40 @@ def conn():
 _tmps = ('tmp%d' % i for i in itertools.count())
 
 
+def test_s3_encrypted_upload():
+    test_bucket = os.getenv('ODO_S3_ENCRYPTION_BUCKET')
+    if not test_bucket:
+        pytest.skip('No bucket defined that requires server-side encryption')
```

Comment:
Please refactor this block into a top-level test fixture, something like

```python
@pytest.fixture
def s3_encryption_bucket():
    test_bucket = os.getenv(...)
    # pytest.skip logic if test_bucket is None
    return test_bucket
```

and declare the fixture as an arg to `test_s3_encrypted_upload(s3_encription_bucket)` and  `test_s3_encrypted_multipart_upload`